build/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+## How to Contribute
```

Comment:
i'm not sure if i'm interpreting the `build/` folder in this PR correctly, but was this added directly, as opposed through a git submodule?  Other projects have been adding https://github.com/upbound/build as a git submodule, so that it can be easily versioned/updated/tracked over time.  See this example:

https://github.com/crossplane-contrib/provider-jet-aws/blob/main/.gitmodules